data/avgi/avgi 1 first contact.txt

```diff
@@ -0,0 +1,834 @@
+			`	He pauses, taking a drink of what looks like water. "They said they had found a gateway of sorts, an ancient alien artifact built by an unknown Predecessor species. Somehow, they had activated it, and that's where the first Aberrant came from."`
+			choice
+				`	"And you want to use this gateway as a way out?"`
+				`	"Where do you think this gateway leads?"`
+			`	"I don't know where it leads, but it'll be somewhere other than Avgi space. Maybe somewhere your jump drive can work, which would let us eventually make our way back to human space. But it's better than sitting here and twiddling our thumbs for another six years."`
+			`	"Where is this gateway?" Darius asks.`
+			`	"The Aberrant came from the East, beyond the furthest reaches of Avgi space. A place the Avgi always had trouble exploring, due to strange energy storms periodically ionizing entire systems. But it's apparently recognizable by the red nebula suffusing it, just as the Twilight is surrounded by a cluster of blue nebulae."`
+			`	"There's no point in waiting, then. If there's a chance, even a small one, we should check it out," says Darius, looking hopeful for the first time since you've met him.`
+
+mission "Avgi: Twilight Escape"
+	name "Escape the Twilight"
+	description "Find a way out of the Twilight. You may want to look to the far east, where the first invasion started."
+	source "Outpost Tekis"
+	passengers 3
+	blocked "You'll need three free bunks on the <ship> to transport Darius, Sora, and Leon."
+	destination "Valhalla"
+	to offer
+		has "Avgi: Dissonance Outreach: done"
+	on enter "Decet"
+		conversation
+			`	Just as your jump drive reacted strangely when you first entered Avgi space, it does so again, but with an inverse effect. Your instrumentation say it is much warmer than usual, almost hot enough to be dangerous. Perhaps your jump drive's range has changed as well.`
+	on enter
+		system
+			attributes "core" "deep" "dirt belt" "near earth" "rim" "south"
+		conversation
+			`	As the purple flames of your jump drive fade, Darius, Sora, and Leon are all speechless. For the first time in six years, they are back in human space.`
+			`	"<first>," says Darius in a shaky voice. "Take us to Valhalla, please. I need to see my home."`
+	on complete
+		clear "avgi: twilight trapped"
+		"reputation: Avgi (Dissonance Aggressive)" <?= -10
+		conversation
+			`	You land in Valhalla's main spaceport, and Darius practically jumps out of your ship before it can even touch down. "I'll be in contact, <first>, but I have to tell someone that I'm still alive."`
+			`	Now that you know of a way both in and out of the Twilight, you can visit the Avgi without fear of being trapped again.`
```

Comment:
Do you plan on including an alternative once the Successors are in, so that one can take the path out that leads through the Tangles via Eta Carinae? Or should that be left up to the player? Additionally, any provisions for someone with metaknowledge of either path taking a beeline out?

Reply:
...Wait, okay, this mission probably runs fine if you go via Eta Carinae.